<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminStati</class>
 <widget class="QDialog" name="AdminStati">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdminStati</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 63)</string>
    </property>
   <property name="text">
    <string/>
   </property>
   
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>600</height>
    </rect>
   </property>
        <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 63)</string>
    </property>

   <widget class="QTableWidget" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>600</height>
     </rect>
    </property>

   </widget>
  </widget>

  <widget class="QPushButton" name="pushButton_leave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 150);
border-bottom:2px solid rgba(105, 118, 132, 255);
color:rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Обратно</string>
   </property>
  </widget>

  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 63);
color:rgba(255, 255, 255, 230);
</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>

  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 63);
color:rgba(255, 255, 255, 230);

</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
